src/config_parser.h

```diff
@@ -91,7 +91,7 @@ class Parameter_t
     MaxConcurrentIO = 10;
     MinSnapshotIndex = 0;
     MinNumPartOfSub = 20;
-    MinNumTracerPartOfSub = 20;
+    MinNumTracerPartOfSub = 10;
```

Comment:
We originally agreed to set this default value to 10.

Reply:
I'm not sure where I got 20 from. 10 is fine by me.